doc/release/technotes/README.chpldoc

```diff
@@ -31,9 +35,9 @@ Generating docs
 
 To create chpldoc documentation, you can either add the '--doc' flag
 to your normal Chapel compiler line or use the standalone 'chpldoc'
-utility.  The primary difference between the two approaches is that
-the former will compile the Chapel program as usual while the latter
-will exit after creating the documentation.
+utility.  As of the 1.11 release, there is no difference between these
+approaches.  It is hoped that the ability to additionally generate an
+executable will be restored soon.
```

Comment:
I'd just drop these three lines starting with "As of..."  In part because I don't think it's a big hardship to not generate an executable and docs at the same time.  In part because I don't think anyone outside the group will care and I don't want the documentation to have to track our hopes over time.